src/Sulu/Bundle/MediaBundle/Resources/js/views/MediaOverview/MediaOverview.js

```diff
@@ -82,7 +94,7 @@ class MediaOverview extends React.PureComponent<ViewProps> {
     render() {
         return (
             <div>
-                {this.title && <h1>{translate(this.title)}</h1>}
+                <h1>{this.getTitle(this.title)}</h1>
```

Comment:
Since we are already using `this` in the `getTitle` function I would argue that you don't need to pass `this.title` to it, but just use it in the function instead.